packages/design-system/src/components/reset/resetDisclosureEls.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default {
+  title: 'Reset / Disclosure Elements',
+};
+
+export const DisclosureElements = () => {
```

Comment:
I plan on adding a `dialog` example here, but with it being a more complex component I didn't want to hold up the reset review

Reply:
@pwolfert I'm actually reconsidering adding reset styles for `details` and `dialog` - mainly for the same thing you pointed out with the form elements. We have components/classes that accomplish all the styles needed for these tags and it doesn't really feel like a reset (`background-color`, `padding`, `border`, etc.). Do you think there's a need for resets on these two tags?

Reply:
No, I'd agree that it's in the same boat and that our other styles cover it. I doubt teams are using bare dialogs that they're resetting themselves. If the goal is for them to be able to trade their resets for ours so theirs doesn't conflict with ours, I think this is an area where they probably don't need a reset of their own.

Reply:
Removed disclosure styles and Story as it overstepped bounds of reset